src/i_sound.c

```diff
@@ -86,6 +86,7 @@ extern char *music_pack_path;
 extern char *fluidsynth_sf_path;
 extern char *timidity_cfg_path;
 #ifdef _WIN32
+extern int winmm_midi_device;
```

Comment:
No `extern` in .c files please. Also this will conflict with #1503 where I'm cleaning up the existing ones.

Reply:
Done.